lib/Connection.js

```diff
@@ -121,6 +131,10 @@ class Connection extends EventEmitter {
       this._closeCallback = null;
     }
     this._ws.removeAllListeners();
+
+    // swallow all WS errors
+    this._ws.on('error', () => {
```

Comment:
I recently debugged a recurring travis failure to this; landed this as https://github.com/GoogleChrome/puppeteer/commit/80078d9526528c0ee7ed745b75595c7274a907f4